desktop/core/src/desktop/conf.py

```diff
@@ -130,6 +130,19 @@ def is_https_enabled():
   """Hue is configured for HTTPS."""
   return bool(SSL_CERTIFICATE.get() and SSL_PRIVATE_KEY.get())
 
+def get_slack_client_id():
```

Comment:
FYI: without a docstring, e.g. """....""" using it as dynamic default might error

Reply:
Is it a best practice to use the docstring wherever possible or it needs to be used in such cases @romainr ?

Reply:
Added the docstrings

Reply:
It is just a special case in Hue, should have got an error (PS: not sure why mandatory, not really needed).

https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/lib/conf.py#L219